frontend/src/metabase/dashboard/components/DashboardActions.jsx

```diff
@@ -8,7 +8,7 @@ import PopoverWithTrigger from "metabase/components/PopoverWithTrigger";
 import RefreshWidget from "metabase/dashboard/components/RefreshWidget";
 import Tooltip from "metabase/components/Tooltip";
 
-export const getDashboardActions = ({
+export const getDashboardActions = (self, {
```

Comment:
I feel gross about this. I tried to use `getDashboardActions.bind(this)(this.props)` where this is used but it didn't work and I maybe don't grok `bind`

Reply:
I agree on the gross feels. Could you extract the `PopoverWithTrigger` code below to its own class-based component?